docs/organize.md

```
@@ -1 +1,102 @@
+
+By specifying usage terms, the license protects the authors' intellectual property while fostering an open-source culture.
+This ensures that everyone understands how to engage with the project responsibly.
+
+By default, all LHM projects are licensed under [MIT](https://choosealicense.com/licenses/mit/).
+
+## Welcome Bot
+
+For new contributors who create their first issue, open their first PR, or merge their first PR, we use a welcome bot to appreciate their contributions.
+The configuration for the bot is inside `.github/config.yml` and can be modified if needed.
+
+The bot is not enabled by default for new projects. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)